### DISCUSSION TITLE
Ne2000 support in ELKS

### DISCUSSION THREAD

**Discussion by kodell-64** [jbruchon/elks#2036] [General]
Hey everyone, loving ELKS here. I've got a 5160 with 8EAISA01.102 Linksys Ether16 16/8 bit ISA network card installed.
I can set the IO and IRQ via DOS utility, ELKS 0.7 detects the card at boot.
I then run the ktcp program with command line arguments. Everything seems fine but I can not telnet, httpget or anything else. netstat reports some transmitted packets but never any received packets.

I've tried ELKS 0.7 with PCEM and ne2k support as well. Same behavior. Any thoughts, pointers? Thanks again. Mangus

**Comment by ghaerr** [comment]
I can't replicate network operations over here on real hardware, but QEMU works with NE2K. We recommend that you run `net start` rather than running ktcp yourself. That then allows you to also `telnet localhost` for testing, although that won't actually touch the network card. The configuration files for networking is in /etc/net.cfg and /bootopts.

**Reply by kodell-64** [reply]
Thanks - yeah I tried all that too. I'll keep at it and will post back if I get it working.

**Reply by kodell-64** [reply]
OK I've got it working with QEMU and PCEM. It's not working with my particular card and it might just be the chipset isn't supported by ELKS. Ive got a 3COM card I can try. Thanks for ELKS ghaerr - really enjoy it.
